packages/cadl-ranch-specs/http/payload/xml/main.tsp

```diff
@@ -130,3 +181,127 @@ interface ModelWithOptionalFieldValue
   </ModelWithOptionalField>
   """
     > {}
+
+@doc("Operations for the ModelWithAttributes type.")
+@route("/modelWithAttributes")
+interface ModelWithAttributesValue
+  extends XmlOperations<
+      ModelWithAttributes,
+      """
+  <ModelWithAttributes id1="123" id2="foo">
+    <enabled>true</enabled>
+  </ModelWithAttributes>
+  """
+    > {}
+
+@doc("Operations for the ModelWithUnwrappedArray type.")
+@route("/modelWithUnwrappedArray")
+interface ModelWithUnwrappedArrayValue
+  extends XmlOperations<
+      ModelWithUnwrappedArray,
+      """
+  <ModelWithUnwrappedArray>
+    <colors>red</colors>
+    <colors>green</colors>
+    <colors>blue</colors>
+    <counts>
+      <int32>1</int32>
+      <int32>2</int32>
```

Comment:
is this the convention to use type name as tag name?

Reply:
Yes, see https://typespec.io/docs/next/libraries/xml/guide

Reply:
i see. thanks for the info.